docs/guides/dws-on-csm/readme.md

```diff
@@ -0,0 +1,185 @@
+---
+authors: Dean Roehrich <dean.roehrich@@hpe.com>
+categories: install
+---
+
+# DWS On CSM: Installation and use of DWS on a CSM cluster
+
+## Install DWS
+
+### DWS Dependencies
+
+Install the DWS dependencies and set the expected node labels prior to installing DWS.
+
+<details>
+<summary>DWS Dependencies</summary>
+    DWS requires kube-rbac-proxy to be present in the cluster's container registry.
+
+```bash
+ncn-m001:~ # podman run --rm --network host quay.io/skopeo/stable:v1.4.1 copy --dest-tls-verify=false docker://gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0 docker://registry.local/kube-rbac-proxy:v0.13.0
+```
+
+DWS will be deployed on a kubernetes worker node labeled with `cray.wlm.manager`.
+
+```bash
+ncn-m001:~ # kubectl label node ncn-w001 cray.wlm.manager=true
+```
+</details>
+
+### Retrieve DWS Configuration and Container Image
+
+The DWS Configuration is in the DWS repo.  We don't need to build DWS here, but we do need its configuration files from the repo.  This is where we will get the DWS CRDs, Deployment, ServiceAccount, Roles and bindings, Services, and Secrets.
+
+```bash
+ncn-m001:~ # DWS_VER=0.0.6
+ncn-m001:~ # git clone --branch v$DWS_VER https://github.com/HewlettPackard/dws.git
+ncn-m001:~ # cd dws
+```
+
+Get the DWS container image corresponding to this repo.  This must be made present in the cluster's container registry.
+
+```bash
+ncn-m001:~/dws # podman run --rm --network host quay.io/skopeo/stable:v1.4.1 copy --dest-tls-verify=false docker://ghcr.io/hewlettpackard/dws:$DWS_VER docker://registry.local/dws:$DWS_VER
+```
+
+### Deploy DWS to the cluster
+
+Deploy DWS to the cluster.  Set `IMAGE_TAG_BASE` to point at the DWS image in the cluster's container registry from the previous step.  Set the `OVERLAY=csm` to pick the configuration for CSM clusters.
+
+```bash
+ncn-m001:~/dws # IMAGE_TAG_BASE=registry.local/dws OVERLAY=csm make deploy
+```
+
+To undeploy DWS, after all Workflow resources have been deleted:
+
+```bash
+ncn-m001:~/dws # IMAGE_TAG_BASE=registry.local/dws OVERLAY=csm make undeploy
+```
```

Comment:
Include the `kubectl wait` command for the dws deployment. That will provide an easy way for the user to check that the deployment was successful and an easy reference to see the name of the deployment.